prow/plugins/assign/assign.go

```diff
@@ -134,12 +134,18 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
+			if e2 := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponse(ae.body, err.Error())); e2 != nil {
```

Comment:
We can do a better error message than that. Take a look at https://github.com/kubernetes/test-infra/blob/master/prow/plugins/lgtm/lgtm.go#L82 for some ideas.

Note that this is more complex because we may assign a variety of people, only a subset of whom are assigned.